README.md

```diff
@@ -26,6 +26,10 @@ services:
       - "80:80/tcp"
     environment:
       TZ: 'America/Chicago'
+      WEB_UID: '1001'
+      WEB_GID: '1000'
+      PIHOLE_UID: '1001'
+      PIHOLE_GID: '1000'
```

Comment:
I think that as these are advanced variables, they probably shouldn't be in the QuickStart example - I can see it causing confusion and support issues.